ts/src/bingx.ts

```diff
@@ -0,0 +1,2342 @@
+        //          "closeTime": 1672026648425
+        //        }
+        //    }
+        //
+        const data = this.safeValue (response, 'data');
+        return this.parseTicker (data, market);
+    }
+
+    async fetchTickers (symbols: string[] = undefined, params = {}) {
+        /**
+         * @method
+         * @name bingx#fetchTickers
+         * @description fetches price tickers for multiple markets, statistical calculations with the information calculated over the past 24 hours each market
+         * @see https://bingx-api.github.io/docs/swapV2/market-api.html#_9-get-ticker
+         * @param {[string]|undefined} symbols unified symbols of the markets to fetch the ticker for, all market tickers are returned if not assigned
+         * @param {object} params extra parameters specific to the bingx api endpoint
+         * @returns {object} a dictionary of [ticker structures]{@link https://docs.ccxt.com/#/?id=ticker-structure}
+         */
+        await this.loadMarkets ();
+        symbols = this.marketSymbols (symbols);
+        const response = await this.swapV2PublicGetQuoteTicker (params);
+        //
+        //    {
+        //        "code": 0,
+        //        "msg": "",
+        //        "data": [
+        //            {
+        //                "symbol": "BTC-USDT",
+        //                "priceChange": "52.5",
+        //                "priceChangePercent": "0.31",
+        //                "lastPrice": "16880.5",
+        //                "lastQty": "2.2238",
+        //                "highPrice": "16897.5",
+        //                "lowPrice": "16726.0",
+        //                "volume": "245870.1692",
+        //                "quoteVolume": "4151395117.73",
+        //                "openPrice": "16832.0",
+        //                "openTime": 1672026667803,
+        //                "closeTime": 1672026648425
+        //            },
+        //        ]
+        //    }
+        //
+        const tickers = this.safeValue (response, 'data');
+        return this.parseTickers (tickers, symbols);
+    }
+
+    parseTicker (ticker, market = undefined) {
+        //
+        //    {
+        //        "symbol": "BTC-USDT",
+        //        "priceChange": "52.5",
+        //        "priceChangePercent": "0.31",
+        //        "lastPrice": "16880.5",
+        //        "lastQty": "2.2238",
+        //        "highPrice": "16897.5",
+        //        "lowPrice": "16726.0",
+        //        "volume": "245870.1692",
+        //        "quoteVolume": "4151395117.73",
+        //        "openPrice": "16832.0",
+        //        "openTime": 1672026667803,
+        //        "closeTime": 1672026648425
+        //    }
+        //
+        const marketId = this.safeString (ticker, 'symbol');
+        const defaultType = this.safeString (this.options, 'defaultType', 'spot');
+        const symbol = this.safeSymbol (marketId, market, '-', defaultType);
```

Comment:
@AlgoQ if right now fetchTicker only supports swap market, then we should use `swap` as the default type here otherwise we get the wrong symbol ⬇️ 

```
✗ n bingx fetchTicker "BTC/USDT:USDT"
Debugger attached.
2023-06-28T11:25:39.543Z
Node.js: v18.14.0
CCXT v3.1.30
bingx.fetchTicker (BTC/USDT:USDT)
2023-06-28T11:25:44.095Z iteration 0 passed in 1278 ms

{
  symbol: 'BTC/USDT',
  timestamp: undefined,
  datetime: undefined,
  high: 30995,
  low: 30145,
  bid: undefined,
  bidVolume: undefined,
  ask: undefined,
  askVolume: undefined,
  vwap: 30616.659976792107,
  open: 30559.2,
  close: 30324,
  last: 30324,
  previousClose: undefined,
```

Reply:
@AlgoQ yes